### DISCUSSION TITLE
What could be the conflict in the Spanish translation PR I created?

### DISCUSSION THREAD

**Discussion by djismgaming** [open-webui/open-webui#3436] [Q&A] [answered]
Hi all and specifically @Yanyutin753! Only if/when time permits. No rush. 😉

I have here something that's been bugging my mind. You had #2720 closed but later on #2744 merged.

tjbck put the same comment about closing #2720 because of a conflict, as in my last #3430.

If possible, how did you resolve this one on your side so it's not in conflict anymore and #2744 passed and was merged? Only if possible. I want to understand the issue to keep on updating the Spanish language as much as possible. 😊

Thanks for all the contributions!

**Comment by Yanyutin753** [comment, marked as answer]
You can use github desktop or pull a copy of the latest dev branch to your own local, and then submit the code you want pr, check whether there is a conflict, after the conflict with vscode or the corresponding command, resolve the conflict, and then you have merged dev, pr to the author！

**Reply by djismgaming** [reply]
thanks so much! following this was key. 😍
